tools/decoder/Decoder.cs

```diff
@@ -325,6 +326,10 @@ public List<string> FromBondToJSONList(bool outputCompactJson = true, bool flatE
                         outputBuffer.SetLength(0);
                     } while (true);
                 }
+                catch (EndOfStreamException)
```

Comment:
Is the exception happening while reading the event from input buffer, and there is nothing to be read ? Just wondering if there is better way to avoid this exception.

Reply:
Because of infinity loop, after data was already read, it is tried to continue and read data from input buffer, but there is nothing to read.
Unfortunately, I couldn't find a way to check that there is nothing to read, and loop should finish.
Open to hear better solution :)

Reply:
I don't know a better way either, but giving it a thought, won't worry much as the decoder component is only used with test-server/fiddler. Should be fine I guess.